isbn/conf.example.php

```diff
@@ -44,9 +44,9 @@
 //HEBIS = Hessisches BibliotheksInformationsSystem
 //http://www.hebis.de/de/1service/z3950/z39_zugang_index.php
 define('HEBIS_URL',         'tolk.hebis.de:20211/hebis');
-define('HEBIS_USER',        '3950');
+define('HEBIS_USER',        '');//TODO look up
 define('HEBIS_PASSWORD',    '');//TODO look up
-define('HEBIS_SYNTAX',      '');//TODO look up
+define('HEBIS_SYNTAX',      'marc21');
```

Comment:
Currently this combination (user name/password) works, but I just consistently don't want to provide user names and passwords through GitHub. See also the other examples in this files. For some of them one needs an individual PWD which we certainly don't want to share with the world.

Moreover, I guess that there is some reason for HEBIS to not open up that restriction completely (maybe they will change the credentials some time, maybe they would like to distinguish the traffic from different users, maybe they don't want to allow some usage...).